### ISSUE TITLE
Is this project dead? 

### ISSUE THREAD

**Issue by llnk** [mridgers/clink#466]
Hi,

I see there are no activies done in about 1 year. Is this project still up? Do you suggest any other updated altnernatives?

**Comment by ankostis** [comment]
Duplicate of #459.

**Comment by CollinChaffin** [comment]
Geez I hope this guy didn't die or something....hate to even say that but when you 100% of a person's public activity stops at the same time......not only here on Github but his Twitter and others too all March 2017.......would be nice if he would at least post here so we know the status of this project, if he is officially ending development and we should fork it, etc.

**Comment by rgd** [comment]
That is strange but hopefully he's just off the grid on holiday or something.  Martin where are you?

**Comment by garoto** [comment]
I sure hope @mridgers didn't simply got tired of it. Clink just changed how I, an exclusive Windows user, see and interact with the (shittastic) builtin CLI. I have being using tons of ffmpeg, mpv, livestreamer, Doom ports like chocolate, etc. only because recalling commands is as easy as one could wish for.

Surely in the top 2 of my most-valuable-software list for our always gimped Windows (Everything being the other one).

And for that you have my eternal gratitude @mridgers.

**Comment by rgd** [comment]
I agree - I'm finding more and more about clink I like and use.  I hope he's not tired of it too, @garoto and I really hope he's ok, @CollinChaffin.

**Comment by ankostis** [comment]
https://www.wired.com/story/giving-open-source-projects-life-after-a-developers-death/

**Comment by gaining** [comment]
No guys the project isn't dead, but rip mridgers. I didn't know the guy personally but I really appreciate his work.

**Comment by Stanzilla** [comment]
urm, he did not really die afaik, @ankostis just posted a random link about dying open source projects.

**Comment by gaining** [comment]
Alright,  wish him the best wherever he may be. :blush:

**Comment by aniketmeshram** [comment]
Guys don't let this die. Can anyone please takeover this project and continue...

**Comment by gaining** [comment]
Apparently lots of people are continuing this project, https://github.com/mridgers/clink/network

**Comment by rgd** [comment]
I saw the 0.49 and 1.0 releases today - does that mean he's back?  Or was it someone else releasing it on his account?

**Comment by garoto** [comment]
I noticed a bunch (1260!?) of commits from previous months are now showing when (iirc) there was nothing there before. Were those sitting on @mridgers's HDD and are being pushed now with the original date?

Fricking awesome regardless!

**Comment by CollinChaffin** [comment]
@gaining Unfortunately with the mystery flood of old commits from @mridgers in Dec (yet still not a single reply in well over half a year since) this repo still contains the most up to date fixes I've found.  When I look at Stanzilla's fork, the only changes I see are related to the installer and since he hasn't even pulled in the upstream I'm not sure I'd use that one.

Clearly @mridgers didn't die since he committed all those but it would really be nice at this point for him to at least inform the community what is going on and whether we NEED to abandon his repo and move to a new fork.

**Comment by atlasjon** [comment]
Anyone recommend a more updated fork of this project?  It's CLEARLY dead at this point.  Last commit was two years ago :(

**Comment by koppor** [comment]
@ghost The currently actively maintained fork is https://github.com/chrisant996/clink.

**Comment by garoto** [comment]
@koppor

`ghost` is how github names all deleted accounts lol.
